api/v1alpha1/dataset_types.go

```diff
@@ -19,7 +19,8 @@ package v1alpha1
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 type DatasetSpec struct {
-	Dataset DatasetConf `json:"dataset" yaml:"dataset"`
+	Dataset    DatasetConf `json:"dataset" yaml:"dataset"`
+	MultiMount *bool       `json:"multipleMountPoints,omitempty" yaml:"multipleMountPoints,omitempty"`
```

Comment:
josn could just be multiMounts

Reply:
Json tag is user-facing. I feel like multipleMountPoints is more clear?

Reply:
this change is no longer?